tests/conftest.py

```diff
@@ -1226,3 +1228,13 @@ def context_config(
         },
         "backup": {"root": {"hiseqx": "flowcells/hiseqx", "hiseqga": "RUNS/", "novaseq": "runs/"}},
     }
+
+
+@pytest.fixture(name="cg_context")
```

Comment:
this gets you the context.obj, but some commands will also need context itself

Reply:
Yes that is another fixture